host_agent.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+__author__ = "Nash Kaminski"
+__copyright__ = "Copyright 2021, Nash Kaminski"
+__license__ = "MIT License"
```

Comment:
Please remove this too. It should inherit the main license.

Reply:
Changed as requested